valkyrie/lib/valkyrie/indexers/access_controls_indexer.rb

```diff
@@ -7,6 +7,7 @@ def initialize(resource:)
     end
 
     def to_solr
+      return {} unless resource.respond_to?(:read_users)
```

Comment:
Just because there's no use case right now for only having some access controls.